src/puppetlabs/trapperkeeper/services.clj

```diff
@@ -22,6 +22,8 @@
   (service-context [this] "Returns the context map for this service")
   (get-service [this service-id] "Returns the service with the given service id")
   (get-services [this] "Returns a sequence containing all of the services in the app")
+  (call-service-fn [this other-service-id fn-name args default] "Calls named function in other service, returning default if that service is not included.")
+  (service-included? [this service-id] "Returns nil if service does not exist and the service if it does")
```

Comment:
oh, duh.  I hadn't gotten far enough to look at the impl (or, apparently, read the docstring) yet.  how do you feel about the name of that fn?  My brain is kind of trained to assume that a fn that is named with a trailing `?` is a predicate function.